lib/elastic_apm/agent.rb

```diff
@@ -122,13 +122,8 @@ def end_transaction(result = nil)
       instrumenter.end_transaction(result)
     end
 
-    def start_span(name = nil, type = nil, backtrace: nil, context: nil)
-      instrumenter.start_span(
-        name,
-        type,
-        backtrace: backtrace,
-        context: context
-      )
+    def start_span(name = nil, type = nil, context: nil)
+      instrumenter.start_span(name, type, context: context)
```

Comment:
Let’s keep the backtrack argument so the Apis stay the same as far as possible. Makes the multi-link delegation easier to grok. I don’t think this will change anything around your fix as it’ll just be nil and do nothing.

Do you agree?

Reply:
And assign it to `span.original_backtrace` in the instrumenter? I think that would be fine. Might want to switch to `span.original_backtrace ||= caller` above then